packages/app-project/src/screens/StatsPage/components/ByTheNumbers/ByTheNumbersContainer.js

```diff
@@ -0,0 +1,72 @@
+import { DateTime } from 'luxon'
+import { inject, observer } from 'mobx-react'
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+
+import ByTheNumbers from './ByTheNumbers'
+
+function storeMapper (stores) {
+  const { project } = stores.store
+  return {
+    classifications: project['classifications_count'],
+    completedSubjects: project['retired_subjects_count'],
+    launchDate: project['launch_date'],
+    volunteers: project['classifiers_count']
+  }
+}
+
+@inject(storeMapper)
+@observer
+class ByTheNumbersContainer extends Component {
+  getLaunchDate () {
+    const { launchDate } = this.props
+
+    // TODO: Use luxon's `.setLocale()` method for doing date translations. This
+    // requires `full-icu` to be included, and I'm going to hold off on that
+    // until https://github.com/nodejs/node/issues/19214 is merged so we have
+    // access to Docker images including it; the docs call it a faff and we
+    // don't have UI translations atm anyway.
+    // See https://moment.github.io/luxon/docs/manual/formatting.html
+    if (launchDate) {
+      return DateTime.fromISO(launchDate)
+        .toLocaleString(DateTime.DATE_FULL)
```

Comment:
The first argument passed to `toLocalString()` should be the locale.

If we're localising dates, then numbers should be localised with eg. `toLocaleString('en')` too (if they aren't already.)

Reply:
nope, this is a luxon datetime object, so it gets set with `setLocale`

good point on localising numbers, but we'd need to hold off for the same reason (no built-in intl support in node yet)

Reply:
Is that using the browser locale by default? That won't be the same as the locale being used by the page.

Reply:
it is at the moment, but we don't have a language setting in the store yet, so that's going to have to be hooked up in a future PR

Reply:
True. Also that store is a per-user session store?

Reply:
The UI store seems to be the logical place to put it - where the value gets populated from is totally up for grabs (cookie? query param? UPP? etc)

Reply:
I was just wondering if the store is a single variable in memory on the server, or if we have a store for each browser session that's currently making requests. Same problem as storing the session theme, I guess.

Reply:
This has been on my mind while considering how auth might work on the server.

Reply:
I'm using `.toLocaleDateString()` in Node without any problems on #1128 so you might not need luxon here.